source/index.rst

```diff
@@ -7,7 +7,7 @@
    :caption: 地震波
    :hidden:
 
-   seismic-wave/index
+   ray-theory
```

Comment:
I think including `taup` here is enough.
```suggestion
   taup
```

Reply:
If we have too much software, the left side bar would be too long.